### ISSUE TITLE
Potential missing BSM terms in B->X_sgamma::BR@NLO

### ISSUE THREAD

**Issue by dvandyk** [eos/eos#236]
Under RGE evolution there is a substantial mixing of the coefficients c8 and c8' into c7 and c7', respectively. This is compensated by the NLO terms in the QCD factorization matrix elements. Looking at the code, I have the impression that we should modify [these lines](https://github.com/eos/eos/blob/master/eos/rare-b-decays/exclusive-b-to-s-gamma.cc#L207L208) as follows:

original:
```
            complex<double> C1nf_top_perp_left = (-1.0 / QCD::casimir_f) * (
                    (wc.c2() - wc.c1() / 6.0) * memoise(CharmLoops::F27_massive, mu(), 0.0, m_b_PS, m_c_pole) + c8eff * CharmLoops::F87_massless(mu, 0.0, m_b_PS));
            const complex<double> C1nf_top_perp_right = 0.0;
```
modified:
```
            complex<double> C1nf_top_perp_left = (-1.0 / QCD::casimir_f) * (
                    (wc.c2() - wc.c1() / 6.0) * memoise(CharmLoops::F27_massive, mu(), 0.0, m_b_PS, m_c_pole) + c8eff * CharmLoops::F87_massless(mu, 0.0, m_b_PS));
            const complex<double> C1nf_top_perp_right = (-1.0 / QCD::casimir_f) * wc.c8prime() * CharmLoops::F87_massless(mu, 0.0, m_b_PS);
```

@cBobeth Do you concur? Handling the four-quark operators is another problem, but luckily in b->sgamma they are less important than in b->sll.

**Comment by cBobeth** [comment]
I agree with the modification for the new physics contribution from c8', we had not included it before.
I would not expect any changes in sign for B -> Xs gamma when going from unprimed to primed
operators.

But in general: c8 mixes into c7 and c8' into c7', but not vice versa. So in NP models where c8, c8' are
generated at some high scale, they will contribute to c7, c7' at some low scale. But since c7, c7' do not
mix into c8, c8' their NP contribution at some high scale will not generate through RG evolution a c7, c7'
contribution at the low scale. All I wanted to say: Our previous code could be used for NP scenarios where
NP only contributed to c7, c7' at any scale but to c8 only at the low scale, whereas no NP to c8'. 

Does this help you?

**Comment by dvandyk** [comment]
Resolved by #425.
